app/views/transcripts/class_2_national_insurance_changes.scala.html

```diff
@@ -26,6 +26,8 @@
 
     @components.heading("class2NationalInsuranceChangesTranscript.heading", "heading-xlarge")
 
+    <p>@messages("class2NationalInsuranceChangesTranscript.paragraph0")</p>
+
```

Comment:
Could maybe remove spaces between paragraphs to keep it consistent with other transcript files. What's your thoughts?

Reply:
done